azkaban-common/src/main/java/azkaban/executor/container/ContainerizedDispatchManager.java

```diff
@@ -224,6 +248,11 @@ public void shutdown() {
     if (null != this.queueProcessor) {
       this.queueProcessor.shutdown();
     }
+
+    if (containerizedWatch.isPresent()) {
```

Comment:
Is there any need to check property here?

Reply:
Good catch. I will make that change.